learning_journal/learning_journal/forms.py

```diff
@@ -0,0 +1,13 @@
+from wtforms import Form, StringField, TextAreaField, validators
+from wtforms import HiddenField
+
+strip_filter = lambda x: x.strip() if x else None
+
+class EntryCreateForm(Form):
+    title = StringField('title', [validators.Length(min=1, max=255)],
+                        filters=[strip_filter])
```

Comment:
Do the constraints in this match those in your entry model?  Don't just copy-paste code from other places.  Think about what it is doing.